ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -2135,5 +2135,81 @@
         "name": "type-java.svg",
         "mediaType": "image/svg+xml"
     }
-  }
+  },
+  {
+  "name": "CentOS WildFly Swarm",
+  "id": "WildFly Swarm",
+  "creator": "Dharmit Shah",
+  "description": "Eclipse WildFly Swarm Stack on CentOS.",
+  "scope": "general",
```

Comment:
Setting scope to general will show the stack in the restricted list of "Ready-to-go" stacks. Are you sure you want to do this?